geonetwork/content.md

```diff
@@ -12,50 +12,151 @@ The project is part of the Open Source Geospatial Foundation ( [OSGeo](http://ww
+docker pull geonetwork:4
+
+docker network create gn-network
+
+docker run -d --name my-es-host --network gn-network -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.17.15
+docker run --name geonetwork-host --network gn-network -e GN_CONFIG_PROPERTIES="-Des.host=my-es-host -Des.protocol=http -Des.port=9200 -Des.url=http://my-es-host:9200" -p 8080:8080 geonetwork:4
+```
+
+For GeoNetwork 4.2:
 
 ```console
-$ docker run --name some-%%REPO%% -d %%IMAGE%%
+docker network create gn-network
+
+docker run -d --name my-es-host --network gn-network -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.17.15
+docker run --name geonetwork-host --network gn-network -e ES_HOST=my-es-host -e ES_PROTOCOL=http  -e ES_PORT=9200 -p 8080:8080 geonetwork:4.2
+```
+
+To be sure about what Elasticsearch version to use you can check the [GeoNetwork documentation](https://docs.geonetwork-opensource.org/4.4/install-guide/installing-index/) for your GN version or the `es.version` property in the [`pom.xml`](https://github.com/geonetwork/core-geonetwork/blob/main/pom.xml#L1528C17-L1528C24) file of the GeoNetwork release used.
+
+### Elasticsearch configuration
+
+#### Java properties (since version 4.4.0)
+
+Since GeoNetwork 4.4.0 the configuration is done using Java properties passed in the `GN_CONFIG_PROPERTIES` environment variable. Property names used for setting the Elasticsearh connection are:
+
+-	`es.host`: *optional* (default `localhost`): The host name of the Elasticsearch server.
+-	`es.port` *optional* (default `9200`): The port where Elasticsearch server is listening to.
+-	`es.protocol` *optional* (default `http`): The protocol used to talk to Elasticsearch. Can be `http` or `https`.
+-	`es.url`: **mandatory if host, port or protocol aren't the default values** (default `http://localhost:9200`): Full URL of the Elasticsearch server.
+-	`es.index.records` *optional* (default `gn_records`): In case you have more than GeoNetwork instance using the same Elasticsearch cluster each one needs to use a different index name. Use this variable to define the name of the index used by each GeoNetwork.
+-	`es.username` *optional (default empty): username used to connect to Elasticsearch.
+-	`es.password` *optional (default empty): password used to connect to Elasticsearch.
+-	`kb.url` *Optional* (default `http://localhost:5601`): The URL where Kibana is listening.
+
+For example, define this environment variable to pass these values to GeoNetwork:
+
+```yaml
+services:
+  geonetwork:
+    image: geonetwork:4.4.0
+    environment:
+      GN_CONFIG_PROPERTIES: >-
+        -Des.host=elasticsearch
+        -Des.protocol=http
+        -Des.port=9200
+        -Des.url=http://elasticsearch:9200
+        -Des.username=my_es_username
+        -Des.password=my_es_password
+        -Dkb.url=http://kibana:5601
 ```
 
-## Publish port
+#### Environment variables (version older than 4.4.0)
```

Comment:
```suggestion
#### Environment variables (version 4.2.0 and older)
```

Reply:
Not really 4.2.0 and older. It applies to all the 4.2 versions and older. Changing it to `version 4.2 and older`.